From chihau@gmail.com  Thu Feb 25 16:00:08 2010
From: Chihau Chau <chihau@gmail.com>
Date: Thu, 25 Feb 2010 17:40:28 -0300
Subject: Staging: comedi: poc: Adding some KERN_ facility level
To: gregkh@suse.de
Cc: wfp5p@virginia.edu, sgayda2@uiuc.edu, mithlesh@linsyssoft.com, devel@driverdev.osuosl.org, linux-kernel@vger.kernel.org, Chihau Chau <chihau@gmail.com>
Message-ID: <1267130428-14175-1-git-send-email-chihau@gmail.com>


From: Chihau Chau <chihau@gmail.com>

This fixes some coding style issues like include KERN_ facility levels
in some printk() and one trailing whitespace error.

Signed-off-by: Chihau Chau <chihau@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/comedi/drivers/poc.c
+++ b/drivers/staging/comedi/drivers/poc.c
@@ -122,22 +122,21 @@ static int poc_attach(struct comedi_device *dev, struct comedi_devconfig *it)
 	unsigned int iosize;
 
 	iobase = it->options[0];
-	printk("comedi%d: poc: using %s iobase 0x%lx\n", dev->minor,
+	printk(KERN_INFO "comedi%d: poc: using %s iobase 0x%lx\n", dev->minor,
 	       this_board->name, iobase);
 
 	dev->board_name = this_board->name;
 
 	if (iobase == 0) {
-		printk("io base address required\n");
+		printk(KERN_ERR "io base address required\n");
 		return -EINVAL;
 	}
 
 	iosize = this_board->iosize;
 	/* check if io addresses are available */
 	if (!request_region(iobase, iosize, "dac02")) {
-		printk
-		    ("I/O port conflict: failed to allocate ports 0x%lx to "
-					"0x%lx\n", iobase, iobase + iosize - 1);
+		printk(KERN_ERR "I/O port conflict: failed to allocate ports "
+			"0x%lx to 0x%lx\n", iobase, iobase + iosize - 1);
 		return -EIO;
 	}
 	dev->iobase = iobase;
@@ -158,7 +157,7 @@ static int poc_attach(struct comedi_device *dev, struct comedi_devconfig *it)
 	s->insn_bits = this_board->insnbits;
 	if (s->type == COMEDI_SUBD_AO || s->type == COMEDI_SUBD_DO)
 		s->subdev_flags = SDF_WRITABLE;
-	
+
 	return 0;
 }
 
-- 
1.5.6.3

